### PR TITLE
Pass tabindex through to tag-input-form.

### PR DESCRIPTION
**Pull request by dougludlow**
If you set the tabindex on the tag-input component, when a user tabs to the tag-input, it moves focus to the component container, not the actual input. This fix takes the tabindex set on the tag-index, unsets it and then passes it through to the input field.

### CODE REVIEW

**Comment by Gbuomprisco** [comment]
Hi @dougludlow, thanks for the PR, I'm just posting some comments

File: .gitignore [hunk 1]
```diff
@@ -3,8 +3,12 @@
 /npm-debug.log
 /typings/
 /coverage
+/modules/**/*.d.ts
+/modules/**/*.js
+/modules/**/*.js.map
 index.js
 index.js.map
 factories
+yarn.lock
```
Comment (Gbuomprisco) [inline .gitignore]: this file needs to be checked :)

File: modules/components/tag-input.ts [hunk 2]
```diff
@@ -352,6 +360,16 @@ export class TagInputComponent extends TagInputAccessor implements OnInit {
      */
     public inputTextValue = '';
 
+
+    /**
+     * @desc removes the tab index if it is set - it will be passed through to the input
+     * @name tabindexAttr
+     * @type {string}
+     */
+    @HostBinding('attr.tabindex') get tabindexAttr(): string {
```
Comment (Gbuomprisco) [inline modules/components/tag-input.ts]: would it be possible to put ` get tabindexAttr(): string ` on the line below and add the modifier `public`? just to be consistent with the rest

File: .gitignore (hunk 1 above)
Comment (dougludlow) [inline .gitignore]: Whoops, didn't realize it was already included. I'll take that out.

File: modules/components/tag-input.ts (hunk 2 above)
Comment (dougludlow) [inline modules/components/tag-input.ts]: Sure thing. Are you looking for something like this?

```typescript
@HostBinding('attr.tabindex') 
public get tabindexAttr(): { 
    return this.tabindex !== undefined ? '-1' : undefined; 
}
```

Comment (Gbuomprisco) [inline modules/components/tag-input.ts]: yep :)

**Comment by dougludlow** [comment]
Sure, I can add it to the readme as well.

**Comment by dougludlow** [comment]
@Gbuomprisco alrighty, should be good to go!

**Comment by dougludlow** [comment]
@Gbuomprisco are you sure about the yarn.lock? It's not currently in the project from what I can see...

**Comment by Gbuomprisco** [comment]
@dougludlow Yeah - but it's good to have for yean users - I actually had planned on adding it

**Comment by dougludlow** [comment]
Fyi, it looks like addOnBlur isn't working, planning on getting a fix in for that as well...

**Comment by Gbuomprisco** [comment]
@dougludlow yeah issue should be at line 805 in ta-input.ts 
`.filter(() => this.dropdown && this.dropdown.isVisible)`

**Comment by dougludlow** [comment]
@Gbuomprisco interesting, i'll start an issue and we can discuss it there.

**Pull request merged**